### ISSUE TITLE
Alt text for "add new user" needs an update

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4108]
## Area or Component
Static site - add new user page
https://www.simplereport.gov/using-simplereport/manage-users/invite-new-users/

## Quick Summary
The alt text for the first image is "manage users tab selected in SimpleReport", but the actual image is of choosing the settings tab in SimpleReport.

## Accessibility Violation
Short text alternative is not appropriate: https://cdc-devtools.dequecloud.com/rule-help/en/alt-text-short-text-not-meaningful

## Action Requested
Change the alt text to "Settings tab selected in SimpleReport"

## Screenshots (optional)
![Screen Shot 2022-07-28 at 2 37 18 PM](https://user-images.githubusercontent.com/80282552/181641951-75132594-2484-4dcd-be33-013ef2bcecfc.png)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/16udNpwdJ8sltAULnmSIGgkWVQsfuFvcoymP0-s_QwfE/edit#gid=1655325714&range=2:2
